lib/skate/location_search/aws_location_request.ex

```diff
@@ -0,0 +1,89 @@
+defmodule Skate.LocationSearch.AwsLocationRequest do
+  alias Skate.LocationSearch.SearchResult
+
+  @base_arguments %{
+    FilterBBox: [-71.55, 42.05, -70.6, 42.65]
+  }
```

Comment:
question (non-blocking): Would it be worth having this set in config and used from both here and in the `Map.tsx` component? May not be worth it since the values likely won't change much.

Reply:
I second this!

Reply:
Good point - I hadn't pursued that assuming it would be hard given that one part of the code is Elixir and the other is TypeScript, but I can just include it in one of the `data-*` attributes that we render in the main `<div>` container in Phoenix.

Reply:
Addressed in e7508098.